### PR TITLE
PSA Version 1.0

### PR DESCRIPTION
**Pull request by cc10512**
This PR sets the version for the PSA to v1.0.

### CODE REVIEW

File: p4-16/psa/PSA.mdk [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Title : P4~16~ Portable Switch Architecture (PSA)
-Title Note : (working draft)
+Title Note: Version 1.0
 Title Footer: &date;
```
Comment (jafingerhut) [inline p4-16/psa/PSA.mdk]: Do we want to hard-code a particular date like 2018-03-01 in place of `&date;` here, too?  (then change it back after the 1.0 release, as these other changes likely will be).

Comment (jafingerhut) [reply p4-16/psa/PSA.mdk]: Except for that, this looks good to me, and I would recommend please that you merge it in for the release.

**Comment by cc10512** [comment]
pushed and published: https://p4.org/specs/

**Comment by jnfoster** [comment]
Huzzah @jafinger and @cc10512!

**Comment by jafingerhut** [comment]
@cc10512 Thank you, and congrats.  The HTML and PDF links for PSA version 1.0 are giving me 'file not found' errors right now.  Not sure if that is a problem on my end or on the server.

**Comment by cc10512** [comment]
Can you please check again?

**Comment by jafingerhut** [comment]
Works for me now.  Thanks.

**Pull request merged**